lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -678,6 +679,7 @@ module Resources
             ],
             'name': [
               {
+                'id': 'CI-12457994-0',
```

Comment:
just to confirm that this is a real id associated with the related person that is being read/searched.

Reply:
Yes, these ids are real ids